src/components/UserSelector.tsx

```diff
@@ -69,20 +59,27 @@ export const UserSelector: React.FC<Props> = ({
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
         <div className="dropdown-content">
-          {users.map(user => (
-            <a
-              key={user.id}
-              href={`#user-${user.id}`}
-              onClick={() => {
-                onChange(user);
-              }}
-              className={classNames('dropdown-item', {
-                'is-active': user.id === selectedUser?.id,
-              })}
-            >
-              {user.name}
-            </a>
-          ))}
+          {users.map(user => {
+            const {
+              id,
+              name,
+            } = user;
+
+            return (
+              <a
+                key={id}
+                href={`#user-${id}`}
+                onClick={() => {
+                  dispatch(setAuthor(user));
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here